README.md

```diff
@@ -1,4 +1,5 @@
 # openHAB2 Docker Containers
+![](https://github.com/microsoft/openhab/openhab-docker/raw/master/images/openhab.png)
```

Comment:
I think the URL should be something like `https://github.com/openhab/openhab-docker/raw/master/images/openhab.png`